### DISCUSSION TITLE
Vercel Github Deploy && Checks Failed

### DISCUSSION THREAD

**Discussion by Quinlivanner** [vercel/community#5606] [Help]
### Summary

When i git push code to github it will auto deploy on vercel.

All deploy working well before.

But today i change some code,and `npm run dev ` in my location PC,all good.Then i git push to github,vercel star deploy.

Some error occur.But the error not effect code running.Site running well.

<img width="821" alt="截屏2024-01-25 23 37 34" src="https://github.com/vercel/community/assets/68963063/6bb44b68-4bd9-4632-91de-9ea9ebe60a21">
<img width="1261" alt="截屏2024-01-25 23 37 57" src="https://github.com/vercel/community/assets/68963063/eb850f62-5243-4de4-9a56-234f78cb3126">

I try to find some error log in anywhere,but nothing.Who know what happen?

### Steps to Reproduce

code have no problem.but checkly failed.

**Comment by txtyash** [comment]
Facing the same issue. Not sure why. Everything seems to be fine
This is the error:
` should respond with correct status code `
and ` Failed to load resource: the server responded with a status of 401 () `
but everything works fine if the checks are skipped

**Reply by Quinlivanner** [reply]
yep but still no reply for team...

**Reply by txtyash** [reply]
Yea, Unfortunate

**Comment by amyegan** [comment]
Hey @Quinlivanner and @txtyash. Since this looks like a Checkly test failure, you would need to log in there to see more details about what failed. I don't know what is being checked in this case so I don't have any ideas about what might have caused it. 🙇

**Reply by Quinlivanner** [reply]
yep i tried this,but till can't find any errlogs.
